static/js/src/navigation.js

```diff
@@ -160,6 +160,25 @@ function addGAImpressionEvents(target, category) {
   }
 }
 
+addGADownloadImpressionEvents(".js-download-option", "download-option");
+
+function addGADownloadImpressionEvents(target, category) {
+  var t = [].slice.call(document.querySelectorAll(target));
+  if (t) {
+    t.forEach(function (section) {
+      if (!section.classList.contains("u-hide")) {
```

Comment:
The section should not have this class. Why check for it?

Reply:
Copy pasting gone wrong. Fixing.

Reply:
Done